src/AdyenPaymentShopware6.php

```diff
@@ -140,4 +140,6 @@ private function setPaymentMethodIsActive(bool $active, Context $context, string
     }
 }
 
-require_once __DIR__ . '/../vendor/autoload.php';
+if (file_exists(__DIR__ . '/../vendor/autoload.php')) {
```

Comment:
what happens if the file doesn't exist? We need the modules from composer otherwise we will get php warnings at other parts of the code - shouldn't we throw an exception here?

Reply:
What I had in mind was that if the autoload file is not there it is because the plugin was installed through composer like described here https://github.com/Adyen/adyen-shopware6/issues/49#issue-692793542 in which case the dependencies are loaded with the Shopware autoload. What do you think?